backend/molgenis-emx2-rdf/src/main/java/org/molgenis/emx2/rdf/RDFService.java

```diff
@@ -74,7 +74,10 @@ public class RDFService {
   public static final String NAMESPACE_DCTERMS = "http://purl.org/dc/terms/";
   public static final IRI IRI_DATABASE_TABLE =
       Values.iri("http://semanticscience.org/resource/SIO_000754");
-  public static final IRI IRI_DATASET = Values.iri("http://purl.org/linked-data/cube#DataSet");
+  public static final IRI IRI_DATASET_CLASS =
+      Values.iri("http://purl.org/linked-data/cube#DataSet");
```

Comment:
are you sure that we want Dataset as a class? I mean, either stuff is Owl:thing or it is this. But I think it is now both?

Reply:
It can be both, I discussed this with Alexander and he's happy with it.